pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTaskRestletResource.java

```diff
@@ -615,12 +615,15 @@ public Map<String, String> scheduleTasks(@ApiParam(value = "Task type") @QueryPa
     tableName = DatabaseUtils.translateTableName(tableName, headers);
```

Comment:
Would it also make sense to extend this API to take in `minionInstanceTag`? 
I understand that `tasks/execute` API already provides that capability. 


We plan to use this feature as follows: 
* Say all minion periodic tasks are usually executed in the minion_untagged hosts.  
* If our minion purge tasks are running behind for a table, we plan to deploy new set of hosts with a special tag e.g. `minion_emergency`, and use the schedule/execute API for one-off scheduling of these jobs on these hosts.

Reply:
Hmm this is a good suggestion. Let me do it in a follow-up as this patch has been open from quite some time. Tracking improvements in #12698. Will add this one there.